aspnetcore/fundamentals/host/generic-host.md

```diff
@@ -0,0 +1,416 @@
+---
+title: .NET Generic Host
+author: guardrex
+description: Learn about the Generic Host in .NET, which is responsible for app startup and lifetime management.
+manager: wpickett
+ms.author: riande
+ms.date: 05/14/2018
```

Comment:
Please add the following metadata:
```
ms.custom: mvc
```

Also, is this doc only applicable to ASP.NET Core 2.1+? If so, add the metadata for that too.